0.10/README.md

```diff
@@ -90,18 +90,52 @@ file inside your source code repository.
 
 Example: DATABASE_USER=sampleUser
 
-Hot deploy
+Development Mode
 ---------------------
-Hot deploy is not supported in this image. To support it, make sure that modules responsible for the server reload are installed upon the build process. Those modules are:
+This image supports development mode. This mode can be switched on and off with the environment variable `DEV_MODE`. `DEV_MODE` can either be set to `true` or `false`.
+Development mode supports two features:
+* Hot Deploy
+* Debugging
+
+The debug port can be speicifed with the environment variable `DEBUG_PORT`. `DEBUG_PORT` is only valid if `DEV_MODE=true`.
+
+A simple example command for running the docker container in production mode is:
+```
+docker run --env DEV_MODE=true my-image-id
+```
+
+To run the container in development mode with a debug port of 5454, run:
+```
+$ docker run --env DEV_MODE=true DEBUG_PORT=5454 my-image-id
+```
+
+To run the container in production mode, run:
+```
+$ docker run --env DEV_MODE=false my-image-id
+```
+
+By default, `DEV_MODE` is set to `true`, and `DEBUG_PORT` is set to `5858`.
```

Comment:
i'm not convinced we want dev_mode on by default, or to be exposing a debug port by default.